### PR TITLE
[RW-6336][risk=no] WGS BQ dataset field / hasWgsData

### PR DESCRIPTION
**Pull request by als364**
Screen Shot 2021-03-03 at 10 28 41 AM

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/config/cdr_config_local.json [hunk 1]
```diff
@@ -99,6 +99,24 @@
...
+    {
+      "cdrVersionId": 6,
+      "isDefault": false,
+      "name": "Synthetic Dataset v3 with Microarray and WGS",
+      "dataAccessLevel": 1,
+      "bigqueryProject": "fc-aou-cdr-synth-test",
+      "bigqueryDataset": "SR2019q4r3",
+      "microarrayBigqueryDataset": "microarray_data",
+      "wgsBigqueryDataset": "1kg_wgs",
+      "creationTime": "2020-08-26 00:09:51Z",
```
Comment (ericsong) [inline api/config/cdr_config_local.json:112]: nit: update to current date

File: api/db/changelog/db.changelog-151-cdr-version-wgs-field.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="als364" id="changelog-151-cdr-version-wgs-field">
+    <addColumn tableName="cdr_version">
+      <column name="wgs_bigquery_dataset" type="varchar(100)">
```
Comment (ericsong) [inline api/db/changelog/db.changelog-151-cdr-version-wgs-field.xml:9]: nit: I think you can drop the `</column>` and just add a `/` to this line

**Review by ericsong — COMMENTED** [review]
Looks good but there's something I forgot to mention in the AC. Can you search the codebase for "RW-6336" and swap out the code where indicated?

File: api/config/cdr_config_local.json (hunk 1 above)
Comment (als364) [reply api/config/cdr_config_local.json:112]: oh lol

**Review by ericsong — APPROVED** [review]
thanks for the changes. There's one more spot that I found but good to go after. https://github.com/all-of-us/workbench/blob/66665eccf926b81c85012b042adc76bc9a588620/ui/src/app/cohort-search/overview/overview.component.tsx#L427

**Pull request merged**